config/.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+      - id: black
+        args: [-S, --line-length=100]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: flake8
+        args: ['--max-line-length=100', '--ignore=F401, W503']
+  - repo: local
+    hooks:
+      - id: version_increase
+        name: version_increase
+        description: Increase version in constants.json
+        entry: env/bin/python config/increase_version.py
```

Comment:
Yeah, that's not very nice indeed.
I now added a check that it only increases the version, when the branch is development.

Furthermore, the local version is only set the the remote version (from pypi) + 0.0.1. So, it will be increased at most once per merge to master.

Reply:
Aaah. That's quite smart! Ignore my other comment, this makes sense!